optimade/server/mappers/entries.py

```diff
@@ -1,10 +1,11 @@
+import abc
 from typing import Tuple
 from optimade.server.config import CONFIG
 
-__all__ = ("ResourceMapper",)
+__all__ = ("BaseResourceMapper",)
 
 
-class ResourceMapper:
+class BaseResourceMapper(abc.ABC):
```

Comment:
Still not a fan of this being a meta-class.
This mainly comes from two things:
- There are no abstract methods left to schematizice how the "real" class should create its methods, and
- One could indeed use this mapper, manually set some of its variables on the instance and be off - I think?

Reply:
Fair points, as I said I was just trying to signal that it should be inherited from. Would you be happy leaving the name as `BaseResourceMapper` and just removing the abc?

Reply:
I've done this in baa9de5c69214fbf8315c0a664edb0b719d914f2

Reply:
> Fair points, as I said I was just trying to signal that it should be inherited from. Would you be happy leaving the name as `BaseResourceMapper` and just removing the abc?

Yup, perfectly fine.
You can even add your sentiment of sub-classing it in the very nice doc-string you already put in, in baa9de5